apis/mutations/unversioned/externaldata.go

```diff
@@ -0,0 +1,167 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package unversioned
```

Comment:
Should all of this code exist in frameworks? Next to https://github.com/open-policy-agent/frameworks/tree/master/constraint/pkg/apis/externaldata/v1alpha1 ?

Might make sense for this code not to live there if there is mutation-specific logic, but `unversioned` could be helpful for validation if not.

Reply:
Yea I think that would make sense since they share the same logic for sending requests and receiving responses.

Reply:
Yeah, we can figure out what's actually getting written and move things just prior to commit, if necessary. Easier that way.